stable/mysqldump/values.yaml

```diff
@@ -14,14 +14,21 @@ mysql:
   username: root
   password:
   port: 3306
+  # db for single db backup
+  db: dbname
 
-## set to `now` to get a one time job, or a cronjob schedule like `0 0 * * *`
-## to get a cronjob.
-schedule: "0/5 * * * *"
-# schedule: now
+# use --all-databases
+allDatabases:
+  enabled: true
+  # creates single backup file with all databases
+  SingleSqlFile: false
```

Comment:
Should start with lowercase.

Reply:
Take proper name into account, e.g. singleBackupFile or singleDumpFile

Reply:
done